examples/DST_Europe/DST_Europe.ino

```diff
@@ -0,0 +1,96 @@
+/* European Daylight Saving Time (DST) example */
+
+// The Real Time Clock is kept in local winter time (standard time). This approach is key for the simplicity of this solution.
+// This code is for a DST from Last sunday in March 1:00 (am) local time until Last sunday in October 3:00 (am) local time. 
+// The DST time can be changed if required, by choosing a different hour in the DateTime function (4th parameter).
+
+#include <RTClib.h>   // https://github.com/adafruit/rtclib 
+RTC_DS3231 rtc; 
+//RTC_DS1307 rtc;
+
+#define USEDST  true // Use DST (true or false)
+#define BAUD   57600 // Baud rate for the serial monitor
+
+DateTime now; 
+int lastsec;
+
+DateTime dstclock(DateTime n) { // Return the given (DST adjusted) date and time according to DST settings (for extensive date and time calculations)
+  
+  DateTime b, e;
+  
+  b = DateTime(n.year(), 3, 31, 1, 0, 0); // Begin of DST: set on March 31 1:000 (am)
+  if (n.month() == 3) b = DateTime(n.year(), 3, 31 - b.dayOfTheWeek(), 1, 0, 0); // Begin of DST: adjusted to last sunday in March 1:00 (am) when actual month is March
+  e = DateTime(n.year(), 10, 31, 3, 0, 0); // End of DST: set on October 31 3:00 (am)
+  if (n.month() == 10) e = DateTime(n.year(), 10, 31 - e.dayOfTheWeek(), 3, 0, 0); // End of DST: adjusted to last sunday in October 3:00 (am) when actual month is October
```

Comment:
As I already explained a couple of times, the DST transitions should happen at 02:00 RTC time, not at 01:00 or 03:00.